### PR TITLE
Adds 2.2.0 documentation around error handling

### PR DESCRIPTION
**Pull request by justin-schroeder**
In this PR, documentation for 2.2.0 features like:

- Form input error props
- Form error props
- Form named error binding
- Error handler support

### CODE REVIEW

**Comment by justin-schroeder** [comment]
FYI, live preview won’t work cause it needs to be linked to the PR branch of `vue-formulate`

File: docs/guide/inputs/file/README.md [hunk 1]
```diff
@@ -130,30 +127,68 @@ export default {
...
+```vue
+<FormulateInput
+  type="file"
+  :value="[{ url: '/path/to/document.pdf' }]"
+/>
+```
+
+<demo-input-hydration />
+
+#### Mime types
```
Comment (andrew-boyd) [inline docs/guide/inputs/file/README.md]: NIce! Great work getting these trimmed out

File: docs/guide/validation/README.md [hunk 2]
```diff
@@ -612,6 +612,13 @@ function myRule (context, ...args) {
 }
 ```
 
+:::warning Validation Rule Names
```
Comment (justin-schroeder) [inline docs/guide/validation/README.md]: @andrew-boyd consider:
```suggestion
:::tip Validation Rule Names
```

File: docs/guide/forms/error-handling/README.md [hunk 3]
```diff
@@ -0,0 +1,324 @@
...
+then calls the `yourErrorHandler` and expects an object response with two properties:
+
+```js
+{
+  inputErrors: { fieldName: ['Unknown email'] },
+  formErrors: ['Unknown error ocurred']
+}
+```
+
+The `handle` method then works up some black magic, and sets those values on
```
Comment (justin-schroeder) [inline docs/guide/forms/error-handling/README.md]:
```suggestion
The `handle` method then sets those values on
```

File: docs/guide/inputs/file/README.md [hunk 4]
```diff
@@ -130,30 +127,68 @@ export default {
...
 ## Getting results
 
 When files are added to a file `type` in Vue Formulate the value is automatically
 transformed into an instance of `FileUpload`, a helper class to wrap the [FileList](https://developer.mozilla.org/en-US/docs/Web/API/FileList)
 object. It is recommended that the backend have a common URL where files can be uploaded.
 
+## Setting initial values
+
+Setting the initial value of a form that contains an uploaded file is as simple
+as passing it back the array of objects containing urls. This will populate the
```
Comment (justin-schroeder) [inline docs/guide/inputs/file/README.md]:
```suggestion
as giving it an array of objects containing urls. This will populate the
```

**Comment by justin-schroeder** [comment]
@andrew-boyd, ok I think this branch is ready, but lets wait till 2.2.0 ships out and then we'll make one more push to this branch including the new package as a dependency.

**Pull request merged**